lib/data/projects_manager.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+
+import 'package:protofast/models/project.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class ProjectsManager {
```

Comment:
Is it some sort of repository class, right? Is it a convention to call it like this, or can we rename it on something similar to ProjectRepository? Are we going to have FeatureManager, how do you think?

Reply:
I used this for now to save the state of the projects between screens. Please feel free to remove it or modify it to best suit the data persistence feature.

We do not need to have a FeaturesManager if this class could handle saving and loading everything.